bin/trigger-coverity-scan.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Get current branch name
+current=$(git rev-parse --abbrev-ref HEAD | tr -d "\n")
+# Coverity scan is triggered only for coverity_scan branch
+git branch -D coverity_scan
```

Comment:
I'm not a fan of redirecting stderr to /dev/null but for this specific command it's justified since the branch may not exist which is harmless. No sense showing an error message to the user.